exercises/poker/poker.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Poker do
+  @doc """
+  Given a list of poker hands, return a list containing the highest scoring hand.
+
+  If two or more hands tie, return the list of tied hands in the order they were received.
+
+  Aces can be used in low (A 2 3 4 5) or high (10 J Q K A) straights, but do not count as
+  a high card in the former case.
+
+  For example, (A 2 3 4 5) will lose to (2 3 4 5 6).
+  """
```

Comment:
This is a really complicated domain. I would perhaps like to see more domain concepts defined in the doc. What is a straight? What is a flush? The difficulty level of this problem is high. I think we can do a little of the domain research for the users.